contracts/interface/IIdentity.sol

```diff
@@ -4,4 +4,7 @@ pragma solidity ^0.6.9;
 import "./IERC734.sol";
 import "./IERC735.sol";
 
-interface IIdentity is IERC734, IERC735 {}
+interface IIdentity is IERC734, IERC735 {
+    event ExecutionFailed(uint256 indexed executionId, address indexed to, uint256 indexed value, bytes data);
```

Comment:
```suggestion
```

`ExecutionFailed` is already defined here https://github.com/onchain-id/solidity/blob/a4ed72a0a7e9078ac56dbec205e3a6e72279d8d6/contracts/interface/IERC734.sol#L30.